testss/utils.ts

```diff
@@ -0,0 +1,21 @@
+export const CACHED_COMMUNITY_GUILD_ID = 907350958810480671n;
```

Comment:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/87189679/166097301-6c53a149-d8b6-415d-a7c9-eb728b8358f9.png">
Instead of hard code, consider Deno.env.get("CACHED_COMMUNITY_GUILD_ID")